code/game/objects/random/maintenance.dm

```diff
@@ -107,6 +107,13 @@ something, make sure it's not in one of the other lists.*/
 				prob(2);/obj/item/device/camera,
 				prob(3);/obj/item/device/pda,
 				prob(3);/obj/item/device/radio/headset,
+				prob(2);/obj/item/toy/tennis,
+				prob(2);/obj/item/toy/tennis/red,
+				prob(2);/obj/item/toy/tennis/yellow,
+				prob(2);/obj/item/toy/tennis/green,
+				prob(2);/obj/item/toy/tennis/cyan,
+				prob(2);/obj/item/toy/tennis/blue,
+				prob(2);/obj/item/toy/tennis/purple,
```

Comment:
These touch a polaris file, so this should probably be marked somehow (or moved to a VR file)

Reply:
I think I was on autopilot at that point. I'll add comment brackets for now and maybe split them off later.

The pizzavoucher needed commenting too anyway.